docs/src/main/paradox/scala/http/routing-dsl/source-streaming-support.md

```diff
@@ -1,13 +1,12 @@
-
 <a id="json-streaming-scala"></a>
```

Comment:
This anchor can be removed as nothing uses it. It seems to have survived ab7744735a6e5a79c23481338832d70f95b3347c due to the empty line removed above.